packages/expo-updates/ios/EXUpdates/AppLoader/AppLoaderTask.swift

```diff
@@ -19,7 +18,10 @@ public protocol AppLoaderTaskDelegate: AnyObject {
    * AppLoaderTask proceed as usual.
    */
   func appLoaderTask(_: AppLoaderTask, didLoadCachedUpdate update: Update) -> Bool
-  func appLoaderTask(_: AppLoaderTask, didStartLoadingUpdate update: Update)
+  func appLoaderTask(_: AppLoaderTask, didStartCheckingForRemoteUpdate body: [String: Any])
+  func appLoaderTask(_: AppLoaderTask, didFinishCheckingForRemoteUpdate body: [String: Any])
```

Comment:
I've modified the interface to remove the unneeded parameter in `didStartCheckForRemoteUpdate()`.

I'm leaving the `didFinish` parameter as `[String: Any]` since this protocol is marked as Objective-C compatible, so I can't use a Swift struct here. If it is indeed safe to remove the Objective-C annotation, then we can do this.